concepticondata/conceptlists/Zhang-2019-308.tsv

```diff
@@ -0,0 +1,309 @@
+Zhang-2019-308-20	20	the bear	bear	1820	BEAR	
+Zhang-2019-308-21	21	the bee	bee	665	BEE	
+Zhang-2019-308-22	22	the belly	belly	1251	BELLY	
+Zhang-2019-308-23	23	the boat	boat	1844	BOAT	
+Zhang-2019-308-24	24	the body, person	body, person	1480	BODY	
+Zhang-2019-308-25	25	the body, self	body, self	1480	BODY	
+Zhang-2019-308-26	26	the brain	brain	1583	BRAIN	
+Zhang-2019-308-27	27	the broad bean	broad bean	2574	BEAN OR PEA	I think this one could be regarded as a general term
+Zhang-2019-308-28	28	the brown-and-white bear	brown-and-white bear	1820	BEAR	
+Zhang-2019-308-29	29	the chicken	chicken	1318	CHICKEN	
+Zhang-2019-308-30	30	the child	child	2099	CHILD	
+Zhang-2019-308-31	31	the claw	claw	72	CLAW	
+Zhang-2019-308-32	32	the clothes	clothes	1895	CLOTHES	
+Zhang-2019-308-33	33	the colour, countenance	colour, countenance	1613	COLOR	
+Zhang-2019-308-34	34	the command	command	1998	COMMAND	
+Zhang-2019-308-35	35	the cooked rice	cooked rice	1191	COOKED RICE	
+Zhang-2019-308-36	36	the cow	cow	1007	COW	
+Zhang-2019-308-37	37	to cry (of birds or animals)	cry (of birds or animals)	1839	CRY	
+Zhang-2019-308-38	38	the dark side of the mountain	dark side of the mountain			too specific to map
+Zhang-2019-308-39	39	the deceased mother	deceased mother			too specific to map
+Zhang-2019-308-40	40	the dream	dream	2374	DREAM	
+Zhang-2019-308-41	41	the drop (classifier)	drop (classifier)	2087	DROP (CLASSIFIER)	
+Zhang-2019-308-42	42	be dusk, dark	dusk, dark	2297	DUSK	
+Zhang-2019-308-43	43	the ear	ear	1247	EAR	
+Zhang-2019-308-44	44	the earthworm	earthworm	2350	EARTHWORM	
+Zhang-2019-308-45	45	eight	eight	1705	EIGHT	
+Zhang-2019-308-46	46	the elbow	elbow	981	ELBOW	
+Zhang-2019-308-47	47	be eldest, great	eldest, great			concept only existent in Old Chinese
+Zhang-2019-308-48	48	be evil	evil	3764	BAD OR EVIL	
+Zhang-2019-308-49	49	the excrement	excrement	675	FAECES (EXCREMENT)	
+Zhang-2019-308-50	50	the eye	eye	1248	EYE	
+Zhang-2019-308-51	51	the eyebrow	eyebrow	1173	EYEBROW	
+Zhang-2019-308-52	52	the face	face	1560	FACE	
+Zhang-2019-308-53	53	the farming	farming			
+Zhang-2019-308-54	54	the farmland	farmland	201	FARM	
+Zhang-2019-308-55	55	the father	father	1217	FATHER	
+Zhang-2019-308-56	56	the father's brother	father's brother	1985	PATERNAL UNCLE (FATHER'S BROTHER)	
+Zhang-2019-308-57	57	the father's sister's child	father's sister's child			
+Zhang-2019-308-58	58	the field	field	212	FIELD	
+Zhang-2019-308-59	59	the fire	fire	221	FIRE	
+Zhang-2019-308-60	60	the firewood	firewood	10	FIREWOOD	
+Zhang-2019-308-61	61	five	five	493	FIVE	
+Zhang-2019-308-62	62	to float	float	1574	FLOAT	
+Zhang-2019-308-63	63	the fly	fly (insect)	1504	FLY (INSECT)	
+Zhang-2019-308-64	64	be fresh	fresh	243	FRESH	
+Zhang-2019-308-65	65	the friend	friend	1325	FRIEND	
+Zhang-2019-308-66	66	the fur garment	fur garment			
+Zhang-2019-308-67	67	the garment	garment	1895	CLOTHES	
+Zhang-2019-308-68	68	the grandchild, nephew	grandchild, nephew	1617	GRANDCHILD	
```

Comment:
I would unlink this one too because GRANDCHILD would be too narrow for 'the grandchild, nephew'.